paymentsheet/src/main/java/com/stripe/android/paymentsheet/LinkHandler.kt

```diff
@@ -29,6 +29,10 @@ internal class LinkHandler @Inject constructor(
         if (state == null) return
 
         _linkConfiguration.value = state.configuration
+
+        // trigger component creation on Link handler setup so that it gets
+        // cached in the coordinator.
+        linkConfigurationCoordinator.getComponent(state.configuration)
```

Comment:
this is what prevented the payment options screen to get `emailFlow` updates. `componentFlow` was never created.